src/main/java/com/synopsys/integration/jenkins/scan/global/Utility.java

```diff
@@ -13,8 +15,98 @@ public static FilePath createTempDir(String directoryName) throws IOException, I
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
     }
 
+    public static void deleteDirectory(File directory) {
+        File[] files = directory.listFiles();
+        if (files != null) {
+            for (File file : files) {
+                if (file.isDirectory()) {
+                    deleteDirectory(file);
+                } else {
+                    file.delete();
+                }
+            }
+        }
+        directory.delete();
+    }
+
+    public static void copyRepository(String targetDirectory, String pluginWorkspaceDirectory) {
+        File workspaceDirectory = Utility.stringToFile(pluginWorkspaceDirectory);
+        File targetDir = Utility.stringToFile(targetDirectory);
+
+        File gitDirectory = new File(targetDirectory, ".git");
+
+        if (gitDirectory.exists() && gitDirectory.isDirectory()) {
+            Utility.deleteDirectory(gitDirectory);
+        }
+
+        if (!targetDir.exists()) {
+            targetDir.mkdirs();
+        }
+        try {
+            FileUtils.copyDirectory(workspaceDirectory, targetDir);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static String defaultBridgeInstallationPath() {
+        String defaultInstallationPath = null;
+
+        String os = System.getProperty("os.name").toLowerCase();
+        String userHome = System.getProperty("user.home");
+
+        if (os.contains("win")) {
+            defaultInstallationPath = String.join("\\", userHome, "Downloads", ApplicationConstants.DEFAULT_DIRECTORY_NAME);
+        } else if (os.contains("nix") || os.contains("nux") || os.contains("mac")) {
+            defaultInstallationPath = String.join("/", userHome, "Downloads", ApplicationConstants.DEFAULT_DIRECTORY_NAME);
+        }
+
+        verifyAndCreateInstallationPath(defaultInstallationPath);
+
+        return defaultInstallationPath;
+    }
+
+    public static void verifyAndCreateInstallationPath(String bridgeUnzipPath) {
+        File directory = new File(bridgeUnzipPath);
+        if (!directory.exists()) {
+            boolean created = directory.mkdirs();
+            if (!created) {
+                System.out.println("Failed to create directory: " + bridgeUnzipPath);
+            }
+        }
+    }
+
+    public static FilePath stringToFilePath(String path) {
+        FilePath filePath = new FilePath(new File(path));
+        return filePath;
+    }
+
+    public static File stringToFile(String path) {
+        File file = new File(path);
+        return file;
+    }
+
+    public static File filePathToFile(FilePath filePath) {
+        String filePathString = filePath.getRemote();
+        File file = new File(filePathString);
+        return file;
+    }
+
+    public static void cleanupInputJson(String bridgeInstallationPath) {
+        File file = new File(bridgeInstallationPath, BridgeParams.BLACKDUCK_JSON_FILE_NAME);
```

Comment:
Have to keep in mind that later we may need to delete coverity and polaris json file as well